draft-polli-ratelimit-headers.md

```diff
@@ -261,6 +261,10 @@ This specification allows describing a quota policy with the following syntax:
    quota-comment = token "=" (token / quoted-string)
 ~~~
 
+In each policy,
+quota-policy parameters like `w` and
+quota-comment tokens MUST NOT occur multiple times.
```

Comment:
Maybe be more specific about the fact that this limitation applies only within a `quota-policy`, since we allow for multiple policies in the header?
```suggestion
quota-comment tokens MUST NOT occur multiple times within the same quota-policy.
```